vars/alvariumCreate.groovy

```diff
@@ -1,7 +1,8 @@
 @GrabResolver(name='jitpack.io', root='https://jitpack.io/')
 @Grab("com.google.errorprone:error_prone_annotations:2.20.0") // fixes alvarium import error
-@Grab(group='com.github.project-alvarium', module='alvarium-sdk-java', version='d18f5aeadd') 
-@Grab("org.apache.logging.log4j:log4j-core:2.15.0")
+@Grab(group='org.slf4j', module='slf4j-api', version='2.0.12')
+@Grab(group='com.github.ali-amin', module='alvarium-sdk-java', version='4431682f14') 
```

Comment:
Should we use project-alvarium/alvarium-sdk-java instead of the fork?

Reply:
Yes, you're right. Nice catch!

Reply:
@michaelehab should be addressed in 4b67113.